README.md

```diff
@@ -0,0 +1,11 @@
+## 🚲 weg-li iOS App
+
+With weg-li you can easily report wrong parking cars.  
+Website: [weg-li](https://www.weg-li.de)
+
+### Setup
+
+* Install latest Xcode and dev tools.
+* Install [Bundler](https://bundler.io) with: `gem install bundler`
+* Install ruby dependencies with: `cd Wegli && bundle install` 
```

Comment:
the `cd` step could be made obsolete when pulling the project structure out of the `Wegli` directory. I think this extra dir is not needed

Reply:
That's true!